apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -145,6 +145,16 @@ function() {
 	}
 );
 
+/**
+ * Load common module.
+ */
+add_action(
+	'plugins_loaded',
+	function() {
```

Comment:
I just now notice that there is already an existing pattern here of using anonymous functions for actions in this file. I think this should be avoided for a couple of reasons: `remove_action` can't be used if needed to remove these actions from other places, it's not a common WP core coding convention, and iirc it can make error log harder to read and track down.